app/src/main/res/layout/scene_office_map.xml

```diff
@@ -46,6 +46,7 @@
         android:id="@+id/roomImage"
         android:layout_width="164dp"
```

Comment:
Coś trzeba z tymi valusami zrobić. Czy te wymiary są celowe czy tylko na razie są takie ustawione? Wiem, że to nie w tym commicie było zmieniane, ale trzeba o tym pamiętać. Transition name też można do valusów dać jako string.